x-pack/plugins/cases/server/client/attachments/add.ts

```diff
@@ -195,22 +190,9 @@ const addGeneratedAlerts = async (
       user: userDetails,
       commentReq: query,
       id: savedObjectID,
+      casesClientInternal,
     });
 
-    if (
```

Comment:
These checks were moved into the `CommentableCase` class so they can be done immediately before creating a case but after some validation.